standalone/src/DelphesPythia8Reader.h

```diff
@@ -58,11 +58,17 @@ class DelphesPythia8Reader : public DelphesInputReader {
 
     // jet matching
 #if PYTHIA_VERSION_INTEGER < 8300
+    Pythia8::CombineMatchingInput* combined   = 0;
+    Pythia8::UserHooks*            m_matching = 0;
```

Comment:
```suggestion
    Pythia8::UserHooks*            matching = 0;
```
Can this variable be renamed to not have the `m_` prefix as that implies it to be a member variable, but it is just a local variable in this function.